packages/table/package.json

```diff
@@ -22,22 +22,22 @@
     "@blueprintjs/core": "^1.0.1",
     "classnames": "^2.2",
     "es6-shim": "^0.35",
-    "react": "^15.0.1 || ^0.14"
+    "pure-render-decorator": "1.2.1"
```

Comment:
@adidahiya why the fixed version on this package in table `dependencies`?
core depends on `^1.1`, shouldn't that be enough?